src/app/components/server/Icons.tsx

```diff
@@ -197,7 +197,7 @@ export const EmailIcon = ({
       aria-label={alt}
       aria-hidden={alt === ""}
       xmlns="http://www.w3.org/2000/svg"
-      viewBox="0 0 14 14"
+      viewBox="0 0 15 15"
```

Comment:
Why did this grow?
Might be moot since I think the icons in this file are several different widths and heights, per:
```sh
git grep -Pno 'viewBox=".*?"' src/app/components/server/Icons.tsx | cat

src/app/components/server/Icons.tsx:23:viewBox="0 0 16 14"
src/app/components/server/Icons.tsx:46:viewBox="0 0 16 16"
src/app/components/server/Icons.tsx:69:viewBox="0 0 18 18"
src/app/components/server/Icons.tsx:96:viewBox="0 0 12 12"
src/app/components/server/Icons.tsx:119:viewBox="0 0 13 7"
src/app/components/server/Icons.tsx:139:viewBox="0 0 15 14"
src/app/components/server/Icons.tsx:160:viewBox="0 0 16 16"
src/app/components/server/Icons.tsx:180:viewBox="0 0 14 14"
src/app/components/server/Icons.tsx:200:viewBox="0 0 15 15"
src/app/components/server/Icons.tsx:221:viewBox="0 0 11 14"
src/app/components/server/Icons.tsx:241:viewBox="0 0 13 13"
src/app/components/server/Icons.tsx:267:viewBox="0 0 16 16"
src/app/components/server/Icons.tsx:290:viewBox="0 0 24 24"
```

But since this says EmailIcon and I vaguely recall some *Icon questions above&hellip;

- `<MultipleUsers />` &mdash; `viewBox="0 0 16 16"`
- `<PhoneIcon />` &mdash; `viewBox="0 0 14 14"`
- `<EmailIcon />` &mdash; `viewBox="0 0 15 15"`
- `<LocationPin />` &mdash; `viewBox="0 0 11 14"`